identity-tools/developer_identity/create-dev-identity.sh

```diff
@@ -46,7 +46,7 @@ cli_help() {
     -o <output_directory>         output directory of identity.json (default: './')
     -i <device_id>                edge-core's internal-id (mandatory)
     -w <hw_type>                  hardware version of the gateway, refer configurations section in
-                                  $DEVID_CLI_DIR/radioProfile.template.json#L228 (default: 'rpi3bplus')
+                                  $DEVID_CLI_DIR/radioProfile.template.json#L228 (default: 'cat /proc/device-tree/model')
```

Comment:
Is this suppose to run within the comment, or is this only suppose to be text?

Reply:
Thanks @cmonr, good point. I moved the HW_VERSION variable before the help and used that.